node_pools.tf

```diff
@@ -99,6 +99,98 @@ resource "google_container_node_pool" "node_pool_mt" {
   }
 }
 
+resource "google_container_node_pool" "node_pool_dynamic_pods" {
+
+  provider = google-beta
+
+  # these can't be created or deleted at the same time.
+  depends_on = [google_container_node_pool.node_pool_platform]
```

Comment:
this should be node_pool_mt to make sure we don't try to add or delete two node pools from the cluster at the same time, which GCP does not allow.

Reply:
Done